platform/src/main/java/org/hillview/table/ObjectArrayColumn.java

```diff
@@ -178,4 +179,9 @@ public static ObjectArrayColumn mergeColumns(final IColumn left, final IColumn r
         }
         return merged;
     }
+
+    @Override
+    public long hashCode64(int rowIndex, long seed) {
+        return LongHashFunction.xx(seed).hashLong((long) this.asDouble(rowIndex,null));
```

Comment:
Is the seed going to be different on each invocation of hashCode64?

If not, you should not be invoking "LongHashFunction.xx(seed)" on every call to hashcode64() because it will allocate a new object every time you call it.

If in the lifetime of a column, the seed is going to be fixed, I would just instantiate one instance of LongHashFunction.xx(seed) in the constructor and re-use that instance for all subsequent calls to hashcode64() against that column.

This comment applies to all other uses of hashCode64 too!